bugsnag-android-core/src/test/java/com/bugsnag/android/ConnectivityCompatTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.bugsnag.android
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.os.Build
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.MockitoJUnitRunner
+
+@Suppress("DEPRECATION")
```

Comment:
nit: could the suppression be added to the specific statements/methods using deprecated code?